### PR TITLE
FFaker::NameTH_EN for transcribed Thai names

### PR DESCRIPTION
**Pull request by firedev**

### CODE REVIEW

File: lib/ffaker/name_th_en.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# encoding: utf-8
+
+module FFaker
+  module NameTH_EN
```
Comment (marocchino) [inline lib/ffaker/name_th_en.rb]: We dont need underbar here.
See other modules.
https://github.com/ffaker/ffaker/blob/master/lib/ffaker/identification_es_cl.rb#L6

File: test/test_name_th_en.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+
+require 'helper'
+
+class TestFakerNameTH_EN < Test::Unit::TestCase
```
Comment (marocchino) [inline test/test_name_th_en.rb]: Same above.

**Comment by firedev** [comment]
Here you go, wish I didn't have to re-define methods from the upper module, but I guess that's how it works.

**Comment by marocchino** [comment]
Thanks

**Pull request merged**